obsproject/obs-studio
# Correct logo on Linux

### Description

Install the correct logo assets. Add 128x128, 256x256, 512x512, and scalable variants.

### Motivation and Context

The logo is the identity of the application, so it makes sense to use the correct logo.

Until now, we've been installing the UI/forms/images/obs.png asset as the logo. However, that's not officially the logo we should be using.


### How Has This Been Tested?

 - Install this branch
 - Notice that the logo picked by the system is the correct one, with a grey-to-black gradient

### Types of changes

 - Bug fix (non-breaking change which fixes an issue)

### Checklist:
